cyvlfeat/sift/dsift.py

```diff
@@ -158,8 +161,8 @@ def dsift(image, step=1, size=3, bounds=None, window_size=-1, norm=False,
             raise ValueError('Step must only contain positive integers.')
 
     # Validate window_size
-    if not isinstance(window_size, int):
-        raise ValueError('Window size must be an integer.')
+    # if not isinstance(window_size, int):
```

Comment:
Just delete this?

Reply:
yes, according to https://github.com/vlfeat/vlfeat/blob/master/vl/dsift.h#L97, the dtype of `windows_size` is `double`.